meta-mender-core/recipes-bsp/u-boot/u-boot-mender-common.inc

```diff
@@ -71,7 +71,13 @@ do_provide_mender_defines() {
         MENDER_UBOOT_STORAGE_DEVICE=`get_uboot_device_from_device ${MENDER_STORAGE_DEVICE}`
     fi
 
-    case "${KERNEL_IMAGETYPE}" in
+    if [ -n "${MENDER_KERNEL_IMAGETYPE_FORCE}" ]; then
+        MENDER_KERNEL_IMAGETYPE="${MENDER_KERNEL_IMAGETYPE_FORCE}"
+    else
+        MENDER_KERNEL_IMAGETYPE="${KERNEL_IMAGETYPE}"
+    fi
+
+    case "${MENDER_KERNEL_IMAGETYPE}" in
```

Comment:
Should use `$MENDER_KERNEL_IMAGETYPE` here (remove the curly braces), to avoid confusion with Bitbake variables.